### PR TITLE
feat: Adding reverse share ability to copy the link

### PR DESCRIPTION
**Pull request by pierrbt**
I fetched the token from the backend and then I created the upload link, in order to display a link icon to copy the reverse share's link. I also created a modal to display the link when the context isn't secured. I added this because it was yet impossible to copy the link of the reverse share once it was created. Now you can copy it anytime.

Like this branch is child to the other PR, this may cause some other changes but it should normally works.

This is a preview : 

image

### CODE REVIEW

**Comment by stonith404** [comment]
These 2 PRs are nice additions, thanks! I'll look into it ASAP but as I have my final exams next week it could take some time.

**Comment by pierrbt** [comment]
Ok, np, take your time there is no hurry. Good luck for your exams.

PS : I also looked to make the 0.16.0 release, using the root package.json scripts, but the `conventional-changelog` is not found, even after installing it with npm. Any idea about this ?

File: frontend/src/pages/account/reverseShares.tsx [hunk 1]
```diff
@@ -119,19 +123,19 @@ const MyShares = () => {
                           <Accordion.Panel>
                             {reverseShare.shares.map((share) => (
                               <Group key={share.id} mb={4}>
-                                <Text maw={120} truncate>
-                                  {share.id}
-                                </Text>
+                                <Anchor href={`${appUrl}/share/${share.id}`}>
```
Comment (stonith404) [inline frontend/src/pages/account/reverseShares.tsx:126]: I think it would be better that the link opens in a new tab. What do you think?

**Review by stonith404 — COMMENTED** [review]
Looks nice thanks! Please see my comment above.

File: frontend/src/pages/account/reverseShares.tsx (hunk 1 above)
Comment (pierrbt) [reply frontend/src/pages/account/reverseShares.tsx:126]: Yes indeed

**Comment by pierrbt** [comment]
I've made the requested changes.

**Review by stonith404 — APPROVED** [review]

**Pull request merged**